articles/appliance/index.md

```diff
@@ -2,57 +2,57 @@
 url: /appliance
 ---
 
-## Auth0 Appliance
+## Auth0 Appliance Deployment and Maintenance
 
-The Auth0 appliance can be deployed in Auth0's cloud, in your cloud or in your own datacenter. When deploying on-premises, you are responsible for monitoring and operating the servers that make the Auth0 service.
+The Auth0 appliance can be deployed in either Auth0's cloud, your cloud, or your own datacenter. If deploying on-premises in your own datacenter, you are responsible for the operation and monitoring of the servers that host the Auth0 service.
 
-[This article](/deployment) details the main differences between the 4 deployment models.
+See [Auth0 Deployment Models](/deployment) for details on the main differences between each of the four deployment models.
 
-###Who installs Auth0?
+###Auth0 installation and setup
 
-Initial deployment, setup, configuration, security patches, software upgrades, and overall maintenance are responsibilities of Auth0. These activities happen on your infrastructure (e.g. your network, servers, etc.), therefore Auth0 closely coordinates access to it with your operations teams.
+The initial deployment, setup, configuration, security patches, software upgrades, and overall maintenance of the Auth0 appliance are the responsibilities of Auth0. Since all these activities run on your infrastructure (e.g. your network, servers, etc.), Auth0 must closely coordinate access with your operations team.
 
-###How many servers are needed?
+###Server requirements
 
-This depends on the expected traffic and availability required. In some scenarios, a single node is sufficient. The minimum highly available deployment requires three network load balanced nodes. No special load balancing logic is needed (e.g. no sticky sessions).
+The number of servers required to host the Auth0 appliance depends on the expected traffic and the desired availability. In some scenarios, a single node is sufficient. The minimum requirement for a highly available deployment is three load balanced network nodes. No special load balancing logic is needed (no sticky sessions).
```

Comment:
not sure if "load balanced network nodes" really makes sense here - I would change it to "load balanced nodes"